core/src/main/java/org/lflang/generator/CodeBuilder.java

```diff
@@ -68,7 +68,7 @@ public int length() {
 
   /** Add a new line. */
   public void newLine() {
-    this.pr("");
+    this.pr("\n");
```

Comment:
This looks a little sus. Wouldn't this result in two newlines?

Reply:
Ya that's fair. The method was essentially not doing anything due to the empty string. I changed the pr method to reflect the proper usage.

Reply:
Are you sure that it was not do anything? My understanding is that the `pr` method adds a newline after the argument that you pass to it. See line 88 of this file.